src/components/CamundaDistributions/Helm.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useActiveVersion } from "@docusaurus/plugin-content-docs/client";
+import CodeBlock from "@theme/CodeBlock";
+
+const getVersion = () => {
+  const docsVersion = useActiveVersion();
+  const camundaVersion = docsVersion.label.match(/\d\.\d/)[0];
+  const camundaMinorVersion = camundaVersion.split(".")[1];
+
+  // Starting from the Camunda 8.4 release, we decoupled the Helm chart version and started from version 9.
+  // For more details check the Helm chart version matrix:
+  // https://.camunda.io/docs/reference/supported-environments/#helm-charts-version-matrix
+  if (camundaVersion >= "8.4") {
+    let chartVersion = `${Number(camundaMinorVersion) + 5}`;
+    // Helm --version does not support regex or glob patterns.
+    // So for non-stable versions, we need to set full version name like "13.0.0-alpha".
+    if (["current", "next"].includes(docsVersion.name)) {
```

Comment:
potentially blocking: Is this doing what we think it is doing? 

Based on this line, I would have expected to see `.0.0-alpha` appended in version 8.7, which is the `current` version of the docs. However, [I don't](https://preview.docs.camunda.cloud/pr-5569/docs/self-managed/setup/install/):

<img width="753" alt="image" src="https://github.com/user-attachments/assets/f583b6a3-08f7-4cae-a9ce-9c6848b43f99" />

Am I misunderstanding the code, or is there a different value than `current` for `docsVersion.name` in version 8.7?

Reply:
@pepopowitz, the code is correct syntactically, but semantically, it's probably not.
It works because our current [Docusaurus config](https://github.com/camunda/camunda-docs/blob/fda491452b0f213792caa81156121a90a780609d/docusaurus.config.js#L529) refers to `8.8` as current.

I'm not sure if we will change that in the docs and go back to the normal flow as we had in the past (so 8.7 is current and 8.7 is next, and there is a bug in the current config) or if it's expected to have the current to be 8.8 version.

If it's a bug, I will update the code.

Reply:
```suggestion
    // Docusaurus refers to vNext as "current".
    if (docsVersion.name === "current") {
```

Ah, I see. I didn't recall that Docusaurus refers to vNext as "current". 

If that's the case, I'd like to see us remove the value from this check that will never be matched, `"next"`, as well as a comment to remind us why we are checking for `"current"`.

Reply:
Fixed :heavy_check_mark: